docs/user_guide/searching/default_pages.rst

```diff
@@ -3,3 +3,10 @@ Tripal Content-Specific Search Tools
 
 
 By default, Tripal will provide a search tool for every Tripal content type.  When a new content type is created, a new search tool is automatically created for that tool.
+
+These search tools will be available at ``/data_search/[content label]``, such as ``/data_search/mrna``.
```

Comment:
The search pages are also listed on the `structure > views` page if you have the `views_ui` module enabled.

Reply:
I think that might be worth mentioning?

Reply:
Ah thanks.  I actually dont have it enabled, so i guess thats why i couldnt find it in the menu.